invofi/apps/frontend/src/lib/migrations/002_multisig_transactions.sql

```
@@ -0,0 +1,195 @@
+-- approver). Tighten reads to a per-deployment signer allow-list if the whole
+-- org should not see the queue. See ADR-0006 "Authorization is layered".
+
+alter table pending_transactions  enable row level security;
+alter table transaction_approvals enable row level security;
+
+drop policy if exists "Read pending transactions" on pending_transactions;
+create policy "Read pending transactions"
+  on pending_transactions for select
+  using (auth.uid() is not null);
+
+drop policy if exists "Create pending transactions" on pending_transactions;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)